### ISSUE TITLE
[Bug]: Could not get the INetworkConnection instance for the adapter GUID

### ISSUE THREAD

**Issue by friederschueler** [sqlitebrowser/sqlitebrowser#3780]
### What did you do?

Open DB Browser for SQLite 3.13.1 (16.10.2024) on Windows 10 (amd64) after installing via x86_64 MSI Installer.

### What did you expect to see?

Just a clean startup

### What did you see instead?

Error Message:  Could not get the INetworkConnection instance for the adapter GUID

### DB4S Version

3.13.1

### What OS are you seeing the problem on?

Windows

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by friederschueler** [comment]
See already closed Bug #3774 . Problem with Qt Networking. But it worked flawless for 3.12.X (not sure which Patch version I used)

**Comment by justinclift** [comment]
Yeah, it's likely to be a pain in the butt to figure out why Qt is deciding to play up.  Saying that from the last time we had to look into Qt network stuff, and I'm pretty sure it took months. 😦

**Comment by mgrojo** [comment]
@friederschueler does this analysis of another product of this same error make sense to you? https://github.com/ModOrganizer2/modorganizer/issues/1326#issuecomment-1013805165

If your case is not related to a SOCKS5 proxy, could you try the instructions in this other comment to investigate the problem further? https://github.com/ModOrganizer2/modorganizer/issues/1326#issuecomment-749020543
